test/test_catalogue.db

```diff
@@ -4,6 +4,7 @@
 # TEST1 is based on J0023+0923
 # TEST2 is based on J0026+6320
 # TEST3 is based on J0357+3205
+# TEST4 is based on J0024-7204J
```

Comment:
Pretty sure this should be "J0024-7204O" as in `derived_catalogue.db`

Reply:
Well spotted. I've fixed this now.